Ensure there is a string result from xwininfo to work with.

chuma opened the pull request:
Corrects this error which I was seeing in my logs.  May be a fix or workaround for issue #7 

```
Apr 06 09:30:05 marceline gnome-session[2149]: Gjs-Message: JS LOG: Extension "maximus-two@wilfinitlike.gmail.com" had error: TypeError: str.split(...)[1] is undefined
```

Haven't seen any issues since making this change.

Fedora 21, gnome-shell 3.14.4-2.fc21

chuma committed:
Ensure there is a string result from xwininfo to work with.

Distopico commented:
This also fix, [#12](https://github.com/wilfm/GnomeExtensionMaximusTwo/issues/12)

ptpt committed:
Shell 3.16: show app title in app menu

ptpt committed:
Shell 3.16: show tooltip

desyncr committed:
Fixes black bar on top

Merged fixes from pixel-saver extension back (https://github.com/deadalnix/pixel-saver/pull/11/commits)

Thanks to @matthijskooijman for these changes.

chuma committed:
Merge branch 'shell-3.16' of https://github.com/ptpt/GnomeExtensionMaximusTwo

chuma committed:
Merge branch 'bugfix/black-bar-on-top' of https://github.com/desyncr/GnomeExtensionMaximusTwo
